cfgov/v1/models/caching.py

```diff
@@ -83,3 +91,19 @@ def purge_all(self):
             )
         )
         resp.raise_for_status()
+
+
+@receiver(post_save, sender=Document)
+@receiver(post_save, sender=CFGOVRendition)
+def cloudfront_cache_invalidation(sender, instance, **kwargs):
+    media_base = settings.MEDIA_URL
+    if hasattr(settings, 'AWS_S3_CUSTOM_DOMAIN'):
+        media_base = settings.AWS_S3_CUSTOM_DOMAIN
```

Comment:
What does this do? Is `AWS_S3_CUSTOM_DOMAIN` only used for buildfiles?

Reply:
I think it's only used on production/content--  it's how django storages knows to generate URL's with 'https://files.consumerfinance.gov' https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#cloudfront

Reply:
Correct, it's only used for production — and it appears to be the only way to get the files.consumerfinance.gov FQDN.